showkase-processor/src/main/java/com/airbnb/android/showkase/processor/logging/ShowkaseValidator.kt

```diff
@@ -56,7 +57,10 @@ internal class ShowkaseValidator {
                 throw ShowkaseProcessorException(
                     "The methods annotated with " +
                             "$annotationName can't be private as Showkase won't be able to access " +
-                            "them otherwise.",
+                            "them otherwise. If you'd like to skip this check and ignore the private " +
+                            "previews, kindly pass skipPrivate=true as an annotation processor option." +
+                            "To learn more about how to set this option, read the Showkase README here- " +
+                            "https://github.com/airbnb/Showkase/blob/master/README.md",
```

Comment:
Reminder to add a section to the readme and maybe link directly to the section. I'm not seeing an existing section.

Reply:
I'll only add it after I'm ready for the next release otherwise it will cause confusion